app/pods/components/layout/md-footer/template.hbs

```diff
@@ -1,13 +1,22 @@
 <div class="footer-container">
-  <span><a class="btn btn-xs btn-warning"
+  <span class={{if overPercent "btn btn-xs btn-danger" "btn btn-xs btn-info"}}>
+    {{fa-icon "archive"}}
+    Local Storage = {{storagePercent}}%
```

Comment:
If the tooltip is present, drop the `Local` to save space.

Reply:
fixed